assets/prefabs/structures/mines/minesCorridor/FireNIceRoom.prefab

```diff
@@ -0,0 +1,18 @@
+{
+    "StructureTemplate": {
```

Comment:
You need to do more than just add this component without some details here it won't actually spawn.

See other successful PR's for more info.
[I think this one should be very close to what you're trying to do](https://github.com/Terasology/GooeysQuests/blob/master/assets/prefabs/structures/mines/minesCorridor/minesLake.prefab)